### PR TITLE
Hotfix/editorial review changes

### PR DESCRIPTION
**Pull request by stuartjennings-up**

### CODE REVIEW

File: src/core/templatetags/dict_lookup.py [hunk 1]
```diff
@@ -1,17 +1,19 @@
 from django import template
 from django.core.urlresolvers import reverse
 from django.template.defaultfilters import linebreaksbr
+from django.utils.safestring import mark_safe
 register = template.Library()
 
 import json
 import os
```
Comment (craiga) [inline src/core/templatetags/dict_lookup.py]: Ordering of `import` statements here is a bit all over the place. From https://www.python.org/dev/peps/pep-0008/#imports:

> Imports should be grouped in the following order:
> - standard library imports
> - related third party imports
> - local application/library specific imports
> 
> You should put a blank line between each group of imports.

File: src/editorialreview/views.py [hunk 2]
```diff
@@ -19,233 +19,230 @@
 
 @is_editor
 def add_editorial_review(request, submission_type, submission_id):
-	check = None
```
Comment (craiga) [inline src/editorialreview/views.py:22]: It's difficult to tell what's going on here; can you configure PyCharm to not be so fickle with spaces vs. tabs? If not, spaces are a better choice (regardless of what Silicon Valley says).

Comment (craiga) [reply src/editorialreview/views.py:22]: https://www.youtube.com/watch?v=SsoOG6ZeyUI

File: src/templates/core/proposals/view_proposal.html [hunk 3]
```diff
@@ -84,20 +84,22 @@
...
 <hr>
-  {% if not proposal.date_review_started %} 
-{% if proposal.status = 'accepted' or proposal.status = 'declined'%} {% else %}<a href="{% url 'start_proposal_review' proposal.id %}" class="btn btn-primary btn-round"><i class="fa fa-plus">&nbsp;</i>Start Review Process</a>{% endif %}{% else %}
 
-  <div class="col-md-12 other-sub-heading"><h4>Review</h4>{% if proposal.status = 'accepted' or proposal.status = 'declined'%} {% else %}<a href="{% url 'add_proposal_reviewers' proposal.id %}" class="btn  btn-transparent-white" style="position: absolute;right:20;margin-top:-37px"><i class="fa fa-plus">&nbsp;</i>Add Reviewers</a>{% endif %}</div>
-
-  {% include "core/proposals/proposal_review_block.html" with review_data=proposal.review_assignments.all %}
-  {% include "core/proposals/proposal_editorial_review.html" %}
+<div class="col-md-12 other-sub-heading"><h4>External Review Assignments</h4>
+{% if proposal.status = 'accepted' or proposal.status = 'declined'%}
```
Comment (craiga) [inline src/templates/core/proposals/view_proposal.html]: Is there supposed to be something here? If not, should the logic be negated and the heading be moved inside the `if`?

**Review by craiga — CHANGES_REQUESTED** [review]

File: src/editorialreview/views.py (hunk 2 above)
Comment (stuartjennings-up) [reply src/editorialreview/views.py:22]: Configured. Should have done it a while ago!

Comment (stuartjennings-up) [reply src/editorialreview/views.py:22]: Realised that I didn't actually make any changes to this file, just had to change indents when I temporarily put in a print statement, so nothing to check

**Review by craiga — APPROVED** [review]

**Pull request merged**